src/server/rest/v1/service/ChargingStationService.ts

```diff
@@ -114,6 +116,35 @@ export default class ChargingStationService {
     if (Utils.objectHasProperty(filteredRequest, 'forceInactive')) {
       chargingStation.forceInactive = filteredRequest.forceInactive;
     }
+    if (Utils.objectHasProperty(filteredRequest, 'manualConfiguration')) {
```

Comment:
Here you need only to save the manualConfiguration flag.
The removal of the charging point and set of charger's params is done in the front-end.

Reply:
@LucasBrazi06 
I implemented it the 'front end way' first, but I experienced some trouble when testing this, because if manual configuration is set to true (e.g. via postman) and there are still charge points, the changed connector settings will not be saved in the backend and we also would get trouble when displaying this in the front end.

Reply:
That's because you do not amend the station template with the manual settings done and then apply it. 
You need to: 
+ Fetch the matching template if any; 
+ Modify all required fields in it to match the manual setup; 
+ Enrich the charging station with the template.

Reply:
I would expect the same behavior as the manual change in the UI:
- Front-end
  - Click on manual change
  - Remove charge points and set the connectors data (they are already set in the UI using the charge point at the beginning, simply apply this value to the connector objects)
  - Save the charging stations with the manual config

- Backend
  - If conf is manual and there are charge points -> exception
  - Save the charging station with the connectors in the DB
  - Do not call the enrich template method
- Boot Notif / Job:
  - Bypass technical part of the template with the manual flag of the charging station

If you put back the flag in the UI + save, in the backend you should delete the connectors + save and reboot the charging station (the simplest solution.) -> Template will be applies for charging station and its connectors.

Let me know if you see something wrong with that logic.

Reply:
The goal is to keep the template information. 
The charging station must still be used regarding its capabilities and charge point specification: static limitation, charging profiles, power sharing, etc.
The current endpoint is not able to cope with manual change to charge point, capabilities, etc. 
That's why Tobias is calling the enrichTemplate backend side.

Reply:
The other capabilities (other than technical) are already set for the charging station you switched to manual.
There are no longer any charge points: manual means unknown from the template and it should not be an hybrid because we cannot setup all info in UI as you wrote.
I think we can still use smart charging or use static limitation with such manual config.
Tomorrow we'll maybe add the charge point editable in the UI and complete the puzzle...

Reply:
> The other capabilities (other than technical) are already set for the charging station you switched to manual.
> There are no longer any charge points: manual means unknown from the template and it should not be an hybrid because we cannot setup all info in UI as you wrote.
> I think we can still use smart charging or use static limitation with such manual config.
> Tomorrow we'll maybe add the charge point editable in the UI and complete the puzzle...

Meetings minutes: 

+ Totally separate station configuration edition and handling when template is enabled and template are disabled => no call and tunables values override are needed in `enrichChargingStationWithTemplate()`;
+ Reuse the current template configuration when disabling templating;
+ Dashboard shall build the full manual configuration data to push to the endpoint and endpoint shall not modify it; 
+ Manually configured stations never have their matching template applied.